packages/nx/src/utils/lock-file/npm.spec.ts

```diff
@@ -243,5 +315,33 @@ describe('npm LockFile utility', () => {
     it('should match the original file on stringification', () => {
       expect(stringifyNpmLockFile(parsedLockFile)).toEqual(lockFileV1);
     });
+
+    xit('should prune the lock file', () => {
```

Comment:
NPM V1 lock file only contains information about required and nested dependencies, but not peer dependencies so we cannot rebuild the entire pruned lock file accurately.